### PR TITLE
add keybindings for draw mode changes and trash

### PR DESCRIPTION
**Pull request by chriswhong**
Adds keybindings:

- When editing `delete` will trigger `draw.trash()`, removing a selected feature or vertex
- When not editing, 
- - `m` will change the draw mode to `draw_point`
- - `l` will change the draw mode to `draw_line_string`
- - `p` will change the draw mode to `draw_polygon`
- - `r` will change the draw mode to `draw_rectangle`
- - `c` will change the draw mode to `draw_rectangle`

Also fixes site.css where some merge conflict markers were accidentally committed.

### CODE REVIEW

**Comment by chriswhong** [comment]
Note, if #748 makes it in, we will also need to add a keybinding for the circle mode.

**Comment by wboykinm** [comment]
> Note, if #748 makes it in, we will also need to add a keybinding for the circle mode.

Wanna merge #748 and add that first here?

**Comment by chriswhong** [comment]
@wboykinm This is good to go now, added keybinding for circle mode, also added the shortcuts in the `title` for each draw tool (e.g. 'Draw Rectangular Polygon (r)')

**Comment by wboykinm** [comment]
> added the shortcuts in the title for each draw tool (e.g. 'Draw Rectangular Polygon (r)')

Great thinking.

File: css/site.css [hunk 1]
```diff
@@ -102,11 +102,7 @@ p.intro-hint:hover {
 }
 
 .top .buttons {
-<<<<<<< HEAD
```
Comment (wboykinm) [inline css/site.css:105]: @chriswhong Looks like another merge artifact here.

**Review by wboykinm — APPROVED** [review]
@chriswhong Awesome!

**Pull request merged**